cmd/main_test.go

```diff
@@ -50,6 +53,10 @@ func TestMain(t *testing.T) {
 			name: "analyze-only",
 			args: "--resource-input-file ../pkg/collector/data/json/Example1.json --filename examples/output/analysis-only.txt",
 		},
+		{
+			name: "analyze-only-resources-shorthand-flag",
+			args: "-r ../pkg/collector/data/json/Example1.json --filename examples/output/analysis-only.txt",
+		},
```

Comment:
do not use the same output file for more than one test, (since they rune in parallel )